service/core/client/src/main/scala/com/lightbend/lagom/internal/client/ClientServiceCallInvoker.scala

```diff
@@ -205,7 +206,7 @@ private[lagom] abstract class ClientServiceCallInvoker[Request, Response](
       else Nil
     }
 
-    requestWithBody.withHeaders(requestHeaders ++ contentTypeHeader ++ acceptHeader: _*).execute().map { response =>
+    requestWithBody.withHttpHeaders(requestHeaders ++ acceptHeader: _*).execute().map { response =>
```

Comment:
@ignasi35,

Since `requestWithBody` already has some headers, it could be better to use `addHttpHeaders` here instead of `withHttpHeaders` (which should discard and replace existing headers). Anyway, this could be done when we release a new version of [play-ws](https://github.com/playframework/play-ws).

Moreover, nice to see that the changeset to update Lagom is not that big. ;-)